### ISSUE TITLE
Multiple conflicting sets of iptables rules

### ISSUE THREAD

**Issue by Lukasa** [Metaswitch/calico#17]
When playing with the neutron-calico Juju charm I found my VMs were unable to communicate with each other. After further investigation I found that the problem was that nova-network, which had previously been running on the box, had not removed its iptables rules when it was terminated.

Because of the OpenStack utilities for combining iptables rules, these rules didn't cause conflicts with calico-compute, so they went undetected. However, they were not updated to reflect the new state of the network. This meant that new machines had their traffic blocked by the *nova-network* rules, even when the calico-compute rules would have allowed it through.

I resolved this temporarily by flushing the rules on my system and then restarting calico-compute (to repopulate its rules). This works in the simple case but may not work generally. We need a way to spot whether calico-compute is the only entity managing iptables rules.

**Comment by plwhite** [comment]
This appears to be a duplicate of issue #80, at least partially.
